python/raw_radar_reader/README.md

```diff
@@ -0,0 +1,48 @@
+# Raw Radar Interface
+
+This directory contains tools for interacting with raw radar data. Zendar
+makes available raw ADC samples from our FMCW radars. The example tools
+in this directory allow researchers to easily use this data format.
+
+## Setup
+
+Please follow the setup instructions in `ZendarSDK/python` prior to using
+these tools.
+
+
+## Example Script
+
+```
+python3 ./python/example.py --root <path-to-data> --calibration <path-to-cal-file>
```

Comment:
With the moving around of the file, it should be `python3 example.py` only.

Reply:
Thanks for finding that